executors/node/numberformat.js

```diff
@@ -80,36 +145,75 @@ module.exports = {
         options = none;
       }
     } else {
-      // Check each option for implementation.
+      // Set default maximumFractionDigits with these exceptions
+      if (options['notation'] && options['notation'] == 'compact' ||
+          options['style'] && options['style'] == 'currency' ||
+          options['maximumFractionDigits']) {
+        // Don't set maximumFractionDigits
+      } else {
+        options['maximumFractionDigits'] = 6;  // Default
+      }
+      // ?????
+      if (options['roundingMode'] == undefined) {
+        options['roundingMode'] = 'halfTrunc';
+      }
```

Comment:
Also should probably be in test generation. Could go either way though: do we want to define the "ambient" behavior of roundingMode and maximumFractionDigits for all numberformat tests, or do we just want to specify those behaviors explicitly? I lean toward making them explicit. If they _do_ have defaults, they should adopt the ECMA-402 defaults.